express-zod-api/tests/__snapshots__/documentation.spec.ts.snap

```diff
@@ -3456,6 +3468,11 @@ paths:
               required:
                 - key
                 - str
+            examples:
+              example1:
+                value:
+                  key: 1234-56789-01
+                  str: test
```

Comment:
was this lost in #2546 ?
or is this somehow redundant any why?

Reply:
found it in https://github.com/RobinTail/express-zod-api/pull/2324/files#diff-f00b33b97ecdfa75fea448b1f57f02f5fb396d4874560b3ebca390b544d0bf7aL4729

Reply:
ok, so this is actually a right thing.
it was lost because of switching to `extractObjectSchema` for flattening, but that didn't mix examples on the top level, assuming that they are always in props.
the change to the snapshot was probably not noticed during #2324

Reply:
it's actually strange because they're supposed to be mixed by `getFinalEndpointInputSchema`